library/std/src/thread/mod.rs

```diff
@@ -866,14 +866,6 @@ pub fn sleep(dur: Duration) {
 ///   if it wasn't already. Because the token is initially absent, [`unpark`]
 ///   followed by [`park`] will result in the second call returning immediately.
 ///
-/// In other words, each [`Thread`] acts a bit like a spinlock that can be
```

Comment:
Also removed this line because calling it a _spin_lock is weird because it's blocking, and calling it a regular lock is also weird because it's allowed to be unlocked without being previously locked. I think the rest of the documentation is already sufficient.